sirepo/job_api.py

```diff
@@ -274,8 +278,11 @@ def _run_mode(request_content):
                 request_content.computeJid,
             )
         )
+    _validate_value(m.sbatchCores, int)
+    _validate_value(m.sbatchHours, float, int)
     return request_content.pkupdate(
         jobRunMode=j,
         sbatchCores=m.sbatchCores,
         sbatchHours=m.sbatchHours,
+        sbatchQueue=m.sbatchQueue,
```

Comment:
queue name needs to be validated.

Reply:
That's done in the agent. I assumed that when you said the queue [types could be denormalized in the job_agent](https://github.com/radiasoft/sirepo/pull/2511#discussion_r424773587) you wanted the check there too? Seems a little strange to have the job_api depend on the job_agent.

Reply:
https://github.com/radiasoft/sirepo/pull/2511/commits/579c3fbbfc59f452081b73a6b75a90de77873122

The queue values are hardcoded right now. Should they come from pkconfig?

Reply:
^ That comment was for another one of your comments. Disregard.

Reply:
https://github.com/radiasoft/sirepo/pull/2511/commits/29dd6577c388758d8ee08269639ad663149ae0af